### DISCUSSION TITLE
2023-08-03

### DISCUSSION THREAD

**Discussion by nlom0218** [woowacourse-teams/2023-haru-study#245] [daily_meeting]
### 오늘의 할 일

- 발표 준비(히이로, 노아)

- 프론트엔드
    - 웹접근성, 타이머(룩소)
    - UX 개선(엽토)

- 백엔드
    - 폴링, SSE, 소켓에 대해 논의해보기
    - 개발 서버 구축 시도

### 내일의 할 일

- 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by nlom0218** [comment]
### 체크인

10점 - 몸에 기운이 넘쳐요!

### 체크아웃

10점 - 발표준비가 너무 재밌었요!

**Comment by jaehee329** [comment]
### 체크인
7점 - 잠이 안 와서 많이 못 잤어요 ㅠㅠ

### 체크아웃
8점 - 나름 알찼다

**Comment by woosung1223** [comment]
### 체크인
8점 - 피곤하지만 오늘 기분이 나쁘지 않슴다!!

### 체크아웃
6점 - oauth가 왜 안되는지 모르겟어요.. redirect_url 오류가 왜 뜰까요..

**Comment by yeopto** [comment]
### 체크인

7점 - 그냥 무난 무난하네요

### 체크아웃

7점 - 훅 분리하고 집가야겠어요.

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 아침에 테스트 리팩토링 다 해서 기분은 좋은데 발표 준비 하러갈 생각에 마음이 무겁네요...

### 체크아웃
9점 - 그동안 진행했던 태스크들을 정리하는 느낌이라 좋네요! 얼른 발표가 끝났으면 좋겠습니다.

**Comment by aak2075** [comment]
### 체크인

9점 - 룩소가 빠져서 아쉽네요

### 체크아웃

10점 - OAuth 빠르게 했으나 거의 다 됐는데 뭐가 안되는지 잘 모르겠네요
